docs/operational-playbook.md

```diff
@@ -0,0 +1,145 @@
+# Operational Playbook
+
+This contains a set of guides for how to perform some common operational and troubleshooting tasks. You can browse the playbook using the Table of Contents generated by GitHub in the top-left of the markdown viewer:
+
+<img width="439" alt="image" src="https://user-images.githubusercontent.com/4321880/192892160-57498f54-fe8b-4332-804f-94fa6e9b9405.png">
+
+## Common Tasks
+
+### Testing a code change
```

Comment:
could be nice to add instructions for testing changes to data import lambdas here as well, i.e. deploy and test the lambda (requires deleting data from your sandbox env if you want to test reimport)